bagel/utilities/model_utils.py

```diff
@@ -5,10 +5,11 @@
 import typer
 from pydantic import ValidationError
 
-import bagel.file_utils as futil
 from bagel import models
 from bagel.mappings import ALL_NAMESPACES, NB
 
+from . import file_utils as futil
```

Comment:
```suggestion
from utilities import file_utils as futil
```

Reply:
Pytest fails with this change, seemingly because `utilities` is not recognized as a top-level module, but rather one nested inside `bagel`. The relative import seems helpful here for keeping the imports self-contained from inside the `bagel` package.

Reply:
Aha right, my bad. It would have to be `bagel.utilities` of course.